astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -93,6 +93,8 @@ def simplify_labels(self):
                     self.text[axis][i] = self.text[axis][i][start:]
                     if starts_dollar:
                         self.text[axis][i] = '$' + self.text[axis][i]
+                if self.text[axis][i] == '$$':
```

Comment:
This is an empty maths block right i.e. `$...$` where there is nothing in between, as opposed to the LaTeX for display maths?

Could you add a comment about this just to make it more obvious when reading it?

Reply:
Done.